test/integration/read-property-multiple.spec.js

```diff
@@ -25,16 +25,16 @@ describe('bacstack - readPropertyMultiple integration', function() {
     client.readPropertyMultiple('127.0.0.1', requestArray, function(err, response) {
       var object = utils.propertyFormater(response.values[0].values);
       expect(response.values[0].objectIdentifier).to.deep.equal({type: 29, instance: 31});
-      expect(object[28]).to.deep.equal([{value: 'Collection of lighting presence detector', type: 7}]);
+      expect(object[28]).to.deep.equal([{value: 'Collection of lighting presence detector', type: 7, encoding: 0}]);
```

Comment:
What do you thing: Would it make sense to assume `utf8` by default (I guesss we do that already) and leaving the explicit encodings away here. This would make the changes less noisy and provide 100% compatibility to the old string encoding API.